app/assets/javascripts/admin/components/admin-directory-toggle.js.es6

```diff
@@ -0,0 +1,33 @@
+import { iconHTML } from 'discourse-common/helpers/fa-icon';
+import { bufferedRender } from 'discourse-common/lib/buffered-render';
+
+export default Ember.Component.extend(bufferedRender({
+  tagName: 'th',
+  classNames: ['sortable'],
+  rerenderTriggers: ['order', 'ascending'],
+
+  buildBuffer(buffer) {
+    const icon = this.get('icon');
+    if (icon) {
+      buffer.push(iconHTML(icon));
+    }
+
+    buffer.push(I18n.t(this.get('i18nKey')));
+    const field = this.get('field');
```

Comment:
Before i used two times but then i rewrite the function so agree, we dont need to set a var now.